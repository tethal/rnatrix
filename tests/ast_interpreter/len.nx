fun main() {
    print(len("42"));
    print(len());
}
