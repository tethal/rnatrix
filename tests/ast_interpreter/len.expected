2
<string>:3:11: error: function len expects 1 argument, but 0 were provided
    print(len());
          ^^^^^
